### PR TITLE
Deploy onboarding docs

### PR DESCRIPTION
**Pull request by thisisjofrank**

### CODE REVIEW

**Comment by thisisjofrank** [comment]
merging should be blocked until new onboarding flow goes live

**Review by donjo — CHANGES_REQUESTED** [review]
left a few notes, thanks for taking this up!

File: deploy/manual/index.mdx [hunk 1]
```diff
@@ -13,32 +13,28 @@ machines, powered by the [Deno runtime](/runtime/manual).
...
+1. [From the Deno Deploy dashboard](https://dash.deno.com) click the "New Project" button.
 
-If you already have a Deno project hosted on GitHub, you can immediately import
-it in Deno Deploy. [From the Deno Deploy dashboard](https://dash.deno.com),
-click the "New Project" button and choose the option to "Select a repository".
+2. Connect to your GitHub account and select the repository you would like to deploy.
 Follow the on-screen instructions to deploy your existing application.
 
-## Option 3: Start with a playground
+3. Complete the Project Configuration form. This will create a GitHub action to deploy your project. Give your project a name and select from the optional framework presets. If you are not using a framework, you can set up your build settings using the form.
```
[the remark this answers is not in the document]
Comment (thisisjofrank) [reply deploy/manual/index.mdx]: I've updated the copy, let me know if you think it needs to be more explicit re - if you've not got a built step

**Review by donjo — APPROVED** [review]
lgtm

**Pull request merged**